scenes/cslib_number.txt

```diff
@@ -118,6 +118,52 @@
 *goto min_loop
 
 
+*comment MAX_STAT
+*comment ------------------------------
+*comment Returns the largest of a set of given stats.
```

Comment:
Returns the variable name of the largest of a set of given stats.

Reply:
done